blockchain/chain/src/store/chain_store.rs

```diff
@@ -294,6 +294,10 @@ pub fn latest_beacon_entry<DB>(db: &DB, ts: &Tipset) -> Result<BeaconEntry, Erro
 where
     DB: BlockStore,
 {
+    if std::env::var("IGNORE_DRAND") == Ok("1".to_owned()) {
+        return Ok(BeaconEntry::default());
+    }
```

Comment:
nah this check should happen at the end of checking for drand entries. and can you have the bytes be the same as Lotus for interop reasons (if we want to interop without drand with both clients)